snowflake/repository.go

```diff
@@ -147,6 +147,14 @@ func (repo *SnowflakeRepository) GetDataUsage(ctx context.Context, minTime time.
 				args = append(args, *maxTime)
 			}
 
+			if len(excludedUsers) > 0 {
+				excluded := excludedUsers.Slice()
+				strBuilder.WriteString(fmt.Sprintf(" AND USER_NAME NOT IN (%s)", generatePlaceholders(len(excluded))))
```

Comment:
I think we can replace this with 
```
strBuilder.WriteString(" AND USER_NAME NOT IN (?)")
args = append(args, sf.Array(excluded))
```
In that way their is no need to generate the placeholders. 
(https://pkg.go.dev/github.com/snowflakedb/gosnowflake#hdr-Binding_Parameters_to_Array_Variables)